emp_stash_fill.user.js

```diff
@@ -186,6 +191,10 @@ if (STASH_API_KEY !== null) {
         });
     });
 
+    stashButton.addEventListener("click", function() {
+    window.open(STASH_DEFAULT, '_blank');
```

Comment:
This should be `STASH` instead of `STASH_DEFAULT` so that it will work for non-default stash URLs

Reply:
Changed to STASH